src/components/Login/Login.js

```diff
@@ -57,6 +60,8 @@ export default class Login extends React.Component {
         return <LoginFormPassphrase disabled={loading} onLogin={login} />;
       case TAB_LEDGER:
         return <LoginFormLedger disabled={loading} onLogin={login} />;
+      case TAB_JSONFILE:
+        return <LoginFormJsonFile disabled={loading} onLogin={login} />;
```

Comment:
And name these `TAB_WALLET_FILE`, `LoginFormWalletFile` to match.

Reply:
OK, changed locally